lib/streamlit/elements/media.py

```diff
@@ -261,12 +283,74 @@ def marshall_video(
         _marshall_av_media(coordinates, proto, data, mimetype)
 
 
+def _validate_and_normalize(data: "npt.NDArray[Any]") -> Tuple[bytes, int]:
+    import numpy as np
```

Comment:
my intention here is to import this conditionally to avoid expensive import when not NumPy array is provided to st.audio

similar to this: https://github.com/streamlit/streamlit/blob/ddf7e50f13c26ead7d3c7c1c9211c793afeb62a8/lib/streamlit/runtime/caching/hashing.py#L284

Reply:
That makes sense - let's just comment this so that other readers understand

Reply:
Explanatory comment added